toolkits/global/packages/global-card/scss/10-settings/springer.scss

```diff
@@ -1,10 +0,0 @@
-/**
- * Card
- * Springer branding for component
- */
-
-// Default setting
-@import 'default';
-
-// Springer overrides
-$card--title-font-weight: normal;
```

Comment:
doesn't that setting need to be preserved?

Reply:
Actually not according to @tulipdexter